makefiles/shared.ninja

```diff
@@ -8,7 +8,7 @@ rule copy
   command = cp $in $out
 
 rule create_categories_page
-  command = echo "---\nlayout: boxes\n---\n" > $out
+  command = echo "---\nlayout: boxes\n---\n:doctitle: $title" > $out
```

Comment:
Is this generating the [front matter](https://jekyllrb.com/docs/front-matter/) for the index page? If so, should the key here be `title` rather than `:doctitle` so that it can be used as `page.title`?

Reply:
https://docs.asciidoctor.org/asciidoc/latest/document/title/
I tested this PR locally, and it correctly sets the title for `documentation/index.html`, `documentation/computers/index.html` & `documentation/computers/linux_kernel.html`